bids2nda/main.py

```diff
@@ -162,11 +163,17 @@ def run(args):
         dict_append(image03_dict, 'image_modality', "MRI")
         dict_append(image03_dict, 'scanner_manufacturer_pd', metadata.get("Manufacturer", ""))
         dict_append(image03_dict, 'scanner_type_pd', metadata.get("ManufacturersModelName", ""))
-        dict_append(image03_dict, 'scanner_software_versions_pd', metadata.get("HardcopyDeviceSoftwareVersion", ""))
+        dict_append(image03_dict, 'scanner_software_versions_pd', metadata.get("SoftwareVersions", ""))
         dict_append(image03_dict, 'magnetic_field_strength', metadata.get("MagneticFieldStrength", ""))
         dict_append(image03_dict, 'mri_echo_time_pd', metadata.get("EchoTime", ""))
         dict_append(image03_dict, 'flip_angle', metadata.get("FlipAngle", ""))
         dict_append(image03_dict, 'receive_coil', metadata.get("ReceiveCoilName", ""))
+        dict_append(image03_dict, 'image_slice_thickness', metadata.get("SliceThickness", ""))
+        dict_append(image03_dict, 'photomet_interpret', metadata.get("PhotometricInterpretation", ""))
+
+        plane = metadata.get("ImageOrientationPatient")
+        get_orientation = lambda place: ['Axial','Coronal','Sagittal'][np.argmax(plane[:3])]
+        dict_append(image03_dict, 'image_orientation.',get_orientation(plane))
```

Comment:
after looking at this data -- I think this is not sufficient, and produces incorrect result.  Let's wait for Matthew to reply on #12  (I will buzz him). FWIW here is my code which is bit more detailed but I think I am missing the point that it should be paired with affine or smth. nibabel/orientations.py actually is probably having the right function to use ;-)

```python
def cosine_to_orientation(img_ornt_pat):
    """Deduce slicing from cosines

    From http://nipy.org/nibabel/dicom/dicom_orientation.html#dicom-voxel-to
    -patient-coordinate-system-mapping

    From Section C.7.6.1.1.1 we see that the "positive row axis" is left to
    right, and is the direction of the rows, given by the direction of last
    pixel in the first row from the first pixel in that row. Similarly the
    "positive column axis" is top to bottom and is the direction of the columns,
    given by the direction of the last pixel in the first column from the first
    pixel in that column.

    Let's rephrase: the first three values of "Image Orientation Patient" are
    the direction cosine for the "positive row axis". That is, they express the
    direction change in (x, y, z), in the DICOM patient coordinate system
    (DPCS), as you move along the row. That is, as you move from one column to
    the next. That is, as the column array index changes. Similarly, the second
    triplet of values of "Image Orientation Patient" (img_ornt_pat[3:] in
    Python), are the direction cosine for the "positive column axis", and
    express the direction you move, in the DPCS, as you move from row to row,
    and therefore as the row index changes.

    Parameters
    ----------
    img_ornt_pat: list of float
       Values of the ImageOrientationPatient field

    Returns
    -------
    {'Axial', 'Coronal', 'Sagital'}
    """
    # we do not care about the signs
    img_ornt_pat = np.abs(img_ornt_pat)
    # we need to figure out first leading dimension and the 2nd
    d1 = int(np.argmax(img_ornt_pat[:3]))
    d2 = int(np.argmax(img_ornt_pat[3:]))
    ds = tuple(sorted([d1, d2]))
    try:
        return {
            (0, 1): 'Axial',
            (1, 2): 'Sagital',
            (0, 2): 'Coronal'
        }[ds]
    except KeyError:
        raise RuntimeError(
            "Could not deduce the image orientation of %r with both directions "
            "being detected as %r. Give us a use-case to fine tune"
            % (img_ornt_pat, ds)
        )
```

Reply:
As I'm sure y'all know, the affine from the DICOM tells you the orientation of the image acquisition relative to the patient.

You get the affine from the ImageOrientationPatient, which defines two columns, and your interpretation of the third column.  There are two options for the third column (assuming it's orthogonal to the other two) - the cross product of the two columns you have, or the negative cross product.  If you have the per-slice positions, you can work out which of these applies - when you don't (for 3D images or greater), you've either got to fish these out of the private fields (Siemens Mosaic) or know what you're doing with the multislice DICOM format (I don't know that very well).

Once you've got the affine, you can work out the orientation, with the nibabel orientations code.

Reply:
Thank you @matthew-brett!  
Unfortunately I am not grasping the full complexity of the situation ATM and for now will just improve this PR with my logic.

Reply:
I am taking that back!  I found that my commit message said "(not correct)" so I guess I would indeed need to get a better grasp of the situation ;-)  Will push another tentative solution in a second, which might be a better start (if not a finish ;)).